src/components/consul/detail/ConsulCommentListItem.js

```diff
@@ -241,13 +243,15 @@ export const ConsulCommentListItem = ({ commentItem, onRefresh, replyList, navig
 
 const LikeDissLikeIcon = ({ cachedVotesUp, cachedVotesDown, like, disslike, onPress, color }) => {
   return (
-    <Touchable onPress={onPress} style={styles.iconButton}>
-      <Icon.Like
-        color={color}
-        style={[styles.icon, { transform: disslike && [{ rotateX: '180deg' }] }]}
-        size={normalize(16)}
-      />
-      <RegularText smallest>{like ? cachedVotesUp : cachedVotesDown}</RegularText>
+    <Touchable onPress={onPress}>
+      <View style={styles.iconButton}>
+        <Icon.Like
+          color={color}
+          style={[styles.icon, { transform: disslike && [{ rotateX: '180deg' }] }]}
+          size={normalize(16)}
+        />
+        <RegularText smallest>{like ? cachedVotesUp : cachedVotesDown}</RegularText>
+      </View>
```

Comment:
according to this, does the following from `ConsulDocumentListItem `also need a change?

```
<Touchable
  style={[styles.button, styles.buttonClose]}
  onPress={() => setModalVisible(!modalVisible)}
>
  <Icon.Close color={colors.lightestText} size={normalize(16)} />
</Touchable>
```

Reply:
Yes, I missed this. I'm updating now.

Reply:
92dbcae56ec8279ee67d9b39e7fd53ac74c1a527